src/index.js

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+import './style.css';
```

Comment:
quick note - css file should be named after the js file for eg index.js should have index.scss